src/cf-typography/src/molecules/list.less

```diff
@@ -0,0 +1,105 @@
+
+//
+// Unstyled list modifier
+//
+
+.m-list__unstyled {
+    padding-left: 0;
+
+    list-style-type: none;
+
+    // This is needed for dd elements.
+    .a-list-item {
+        margin-left: 0;
+    }
+}
+
+//
+// Spaced list modifier
+//
+
+.m-list__spaced {
+    .m-list__spaced,
+    .a-list-item + .a-list-item {
+        margin-top: unit( 24px / @base-font-size-px, em );
+    }
+}
+
+//
+// Horizontal list modifier
+//
+
+.m-list__horizontal {
```

Comment:
Removing duplicated code. I know gzipping takes care of some of this, but I've been trying to target duplication where possible so we can get to the cleanest output possible and then work our way back. These lists are used in so few templates, I think it's a good trade-off.